timelock-agent/src/main/java/com/palantir/timelock/config/PaxosInstallConfiguration.java

```diff
@@ -27,8 +27,31 @@ default File dataDirectory() {
         return new File("var/data/paxos");
     }
 
+    /**
+     * Set to true if this is a new stack. Otherwise, set to false.
+     */
+    @JsonProperty("is-new-service")
+    boolean isNewService();
+
     @Value.Check
     default void check() {
+        if (isNewService() && dataDirectory().isDirectory()) {
+            throw new IllegalArgumentException(
+                    "This timelock server has been configured as a new stack (the 'is-new-service' property is set to "
+                            + "true), but the Paxos data directory already exists. Almost surely this is because it "
+                            + "has already been turned on at least once, and thus the 'is-new-service' property should "
+                            + "be set to false for safety reasons.");
+        }
+
+        if (!isNewService() && !dataDirectory().isDirectory()) {
+            throw new IllegalArgumentException(
+                    "If this is a new timelock service, please configure paxos.is-new-service to true for the first "
+                            + "startup only of each node. \n\nOtherwise, the timelock data directory appears to no "
+                            + "longer exist. If you are trying to move the nodes on your timelock cluster or add new "
+                            + "nodes, you have likely already made a mistake by this point. This is a non-trivial "
+                            + "operation, please be careful and do not be afraid to seek out help.");
```

Comment:
I think we can be more direct here and instruct people to speak to the Atlas team before proceeding.

Reply:
i've cited 'appropriate approvals' - in short these messages with a 'who' on the end have typically ended up fairly out of date...